install/installer/pkg/components/ide-service/ide_config_configmap.go

```diff
@@ -195,6 +196,18 @@ func ideConfigConfigmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 					ImageLayers:       []string{jbPluginImage, jbLauncherImage},
 					LatestImageLayers: []string{jbPluginLatestImage, jbLauncherImage},
 				},
+				xterm: {
```

Comment:
Here we add the IDE to the IDE list. Currently, latest is the same as stable, since there are no proper versions yet.